.github/labeler.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Add 'cicd' label to any file changes inside .github dir
+cicd:
+  - .github/*
+  - .github/**/*
+
+# Add 'patterns' label to any file changes for the patterns
+patterns:
+  - source/_patterns/**
+
```

Comment:
I would add "tokens" as label as well, what do you think @mfranzke?

Reply:
loving it !